src/main/java/net/greatstart/controllers/EventController.java

```diff
@@ -0,0 +1,35 @@
+package net.greatstart.controller;
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') This file "EventController.java" should be located in "net/greatstart/controller" directory, not in "/home/travis/build/Sheptytskyid/Startup_Platform/src/main/java/net/greatstart/controllers". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1598)